bin/dump-databases

```diff
@@ -1,2 +0,0 @@
-#!/bin/sh
-mysqldump --databases inbox inbox_1 inbox_2 inbox_3 -uroot -proot
```

Comment:
This just doesnt make sense since our databases are not named like that anymore. And I never used it.